platform.txt

```diff
@@ -127,8 +127,8 @@ build.flash_freq=80m
 build.boot=qio
 build.boot_freq={build.flash_freq}
 build.bootloader_addr=0x1000
-build.custom_bootloader=bootloader
-build.custom_partitions=partitions
+build.custom_bootloader=
+build.custom_partitions=
```

Comment:
This makes the build search for `variant/.bin`, expanded from `"{build.variant.path}"/{build.custom_bootloader}.bin`. Maybe it should be left as is?

Reply:
yeah, this is meant to expanded to non-existed files by default in the variant/board folder so that one of the espressif default parition csv can be used if an TinyUF2 scheme isn't selected.

Reply:
it will still expand to non-existing if you name yours to something else. Default values are there to allow variants to add bootloader.bin and partitions.csv and be done, without having to specify anything in boards.txt. Please rename your bootloaders to something else and the defaults here will work as expected :)

Reply:
update PR to revert this changes and rename the bootloader/partition in the variant folder as suggested.